xain_fl/cli.py

```diff
@@ -197,7 +222,15 @@ def main():
         fraction_of_participants=parameters.fraction,
     )
 
-    serve(coordinator=coordinator, host=parameters.host, port=parameters.port)
+    store_config = StoreConfig(
+        parameters.storage_endpoint,
+        parameters.storage_key_id,
+        parameters.storage_secret_access_key,
+        parameters.storage_bucket,
+    )
+    store = Store(store_config)
```

Comment:
Maybe it makes sense to create a in-memory Store in a later task and use it as the default store. I just want to make sure that it is still possible to run the coordinator without the requirement of minio. What do you think @atymoshchuk, @little-dude, @wilk10?

Reply:
Agree. I was thinking about the same thing while writing the tests.

Reply:
I created a ticket [XP-435](https://xainag.atlassian.net/browse/XP-435).